index.ts

```diff
@@ -0,0 +1,19 @@
+import { v4 as uuidv4 } from 'uuid';
+
+const createPacket = (data: any): { data: any, id: string} => {
```

Comment:
it's best practice to name our types, please extract the return type to a type with a name :)